### DISCUSSION TITLE
DialogDisclosure missing from Dialog example code

### DISCUSSION THREAD

**Discussion by everdimension** [ariakit/ariakit#1981] [Questions] [answered]
Hi! I'm reading docs for the Dialog component:
https://ariakit.org/components/dialog

In reakit, we used `<DialogDisclosure />` component as a button to open the dialog

I see that in ariakit API this component still exists, but the code example is missing its usage.

Is this just something that was missed or are there reasons for not using it any more?

**Comment by diegohaz** [comment, marked as answer]
`DialogDisclosure` is optional when using `Dialog`. But this component is used in `MenuButton`, `PopoverDisclosure` and other components.

We're not promoting it in the Dialog examples anymore mostly because of https://github.com/ariakit/ariakit/issues/925.
